src/html.js

```diff
@@ -15,6 +15,9 @@ const HTML = (props) => (
       <link href="https://fonts.googleapis.com/css2?family=Rubik&display=swap" rel="stylesheet" />
       <link href="https://fonts.googleapis.com/css2?family=Roboto&display=swap" rel="stylesheet" />
       <link href="https://fonts.googleapis.com/css2?family=Inter&display=swap" rel="stylesheet" />
+      <meta name="description" content="Skyline is an open source and experimental emulator that runs on ARMv8 Android™ devices and emulates the functionality of a Nintendo Switch™ system." />
+      <meta name="keywords" content="Emulator, Nintendo Switch™, Skyline, Android" />
```

Comment:
I think the TM symbol should be removed as it might actually be bad for SEO and isn't legally necessary.